### PR TITLE
Minor sync to what is in production

### PR DESCRIPTION
**Pull request by aidvu**
To get it to work with node v16 and latest qmake.

### CODE REVIEW

File: lib/comms.js [hunk 1]
```diff
@@ -97,6 +97,8 @@ var client = {
 			logger.error( 'timed out performing a request to server ' + options.hostname );
 		};
 
+		options.secureProtocol = "TLSv1_2_method";
```
[the remark this answers is not in the document]
Comment (fgiannar) [reply lib/comms.js:100]: Thanks! Created a [corresponding issue](https://github.com/Automattic/jetmon/issues/7) to track this.

File: src/http_checker.cpp [hunk 2]
```diff
@@ -1,5 +1,6 @@
 
 #include "http_checker.h"
+#include <cstring>
```
Comment (fgiannar) [inline src/http_checker.cpp:3]: Non blocking: Ideally this should be moved to `http_checker.h` header file along with the rest includes.

**Review by fgiannar — APPROVED** [review]
Many thanks @aidvu ! Appreciate your help aligning the repo with production

File: src/http_checker.cpp (hunk 2 above)
Comment (aidvu) [reply src/http_checker.cpp:3]: I was going for "minimal changes to get it to compile". 😆 thanks.

**Pull request merged**